spec/javascript/e2e/ApplicationNewPage.e2e.js

```diff
@@ -1,23 +1,35 @@
-import puppeteer from 'puppeteer'
-
 import utils from '../support/puppeteer/utils'
 import sharedSteps from '../support/puppeteer/steps/sharedSteps'
 import { applicationRedirectRouteCheck } from '../support/puppeteer/steps/applications'
 import {
-  NON_LEASE_UP_LISTING_ID, DEFAULT_E2E_TIME_OUT, HEADLESS, SALE_LISTING_ID, LEASE_UP_LISTING_ID,
+  NON_LEASE_UP_LISTING_ID, DEFAULT_E2E_TIME_OUT, SALE_LISTING_ID, LEASE_UP_LISTING_ID,
   FIRST_NAME, LAST_NAME, TRUNCATED_FIRST_NAME, TRUNCATED_LAST_NAME, DOB_MONTH, DOB_DAY, DOB_YEAR,
   DATE_OF_BIRTH, HOUSEHOLD_MEMBER_FIRST_NAME, HOUSEHOLD_MEMBER_LAST_NAME, HOUSEHOLD_MEMBER_DATE_OF_BIRTH,
   HOUSEHOLD_MEMBER_DOB_MONTH, HOUSEHOLD_MEMBER_DOB_DAY, HOUSEHOLD_MEMBER_DOB_YEAR, LENDING_INSTITUTION, LENDING_AGENT_ID
 } from '../support/puppeteer/consts'
+import SetupBrowserAndPage from '../utils/SetupBrowserAndPage'
+let testBrowser
 
 describe('ApplicationNewPage', () => {
-  test('should create a new application successfully', async () => {
-    let browser = await puppeteer.launch({ headless: HEADLESS })
-    let page = await browser.newPage()
+  test('should fail if required fields are missing, and create application if not', async () => {
+    let { browser, page } = await SetupBrowserAndPage()
+    testBrowser = browser
```

Comment:
I am not sure, I will check to see if there is a way to handle that cleaner

Reply:
I looked into a few options but I am not sure that our current setup supports it and it may take another decent chunk of time to refactor to a `beforeAll/beforeEach` approach with something like `jest-puppeteer`

Reply:
ok thanks for looking into it